bsp/boards/OpenMote-CC2538/board_info.h

```diff
@@ -66,9 +83,10 @@
 #define PORT_maxRxDataPrepare               33    // 1007us (measured  84us)
 #define PORT_maxTxAckPrepare                22    //  305us (measured 219us)
 // radio speed related
-#define PORT_delayTx                        7     //  214us (measured 219us)
+#define PORT_delayTx                        12    //  214us (measured 219us)

```

Comment:
Yes, there is something I need to say here. 12 is the right delayTx value for OpenMote-CC2538. With 12, OpenMote-CC2538 can synchronize perfectly.

When security enabled with 10 ms slot timing, the interval from received a frame to send back ack is not long enough for decryption of income packet and encrytion of ACK if the frame is long, like over 100 bytes. 

Setting delayTx smaller, will give some space for mote to processing long frame when security enabled with 10ms slot. However, the cost is we lost some accuracy on synchronization.